### ISSUE TITLE
No way to see if someone is emoting at you

### ISSUE THREAD

**Issue by MayRedBeWithYou** [Xpahtalo/Ffxiv2Mqtt#10]
As in the title, I would love to have automation based on if someone /dotes on me etc. :)

**Comment by Xpahtalo** [comment]
As this would require extra work outside of what is provided by Dalamud, I haven't decided if I want to include it yet. I'm trying to limit how much custom stuff is going on so that it "just works". The version that will start shipping with 6.2 is easier to work with IPC, so other plugin devs could potentially support it. I can think of a few cute ideas for emote triggers, so I might end up making an exception, but if I do, it's unlikely to be until after a couple weeks in 6.2 due to life.

**Comment by MayRedBeWithYou** [comment]
Makes sense - I hoped that there is some OnEmote event anywhere, but it seems that Dalamud doesn't have that. On the other hand, this is something that's already solved by [PatMe plugin](https://github.com/MgAl2O4/PatMeDalamud) by MgAl2O4 with its EmoteReaderHook. It might be worth looking into (might try it myself) when the time is right 😄
